app/views/common_parameters/_parameter.html.erb

```diff
@@ -3,15 +3,15 @@
   <table class="row">
     <tbody>
       <tr class="form-group <%= 'has-error' if f.object.errors.any? %>">
-        <% if controller_name =~ /(host|hostgroup)/ %>
```

Comment:
Then wrap the header in app/views/common_parameters/_parameters.html.erb in the same check?